lib/zendesk_apps_support/location.rb

```diff
@@ -30,22 +31,24 @@ def self.all
     LOCATIONS_AVAILABLE = [
```

Comment:
This is getting quite difficult to visually parse. Could we setup some defaults on the constructor to reduce the redundancy?